web-console/knikubevirt/VMIs/VMIs.md

```diff
@@ -0,0 +1,66 @@
+# Surfacing VMIs in the UI
+
+VMI is the current running VM.
+The initial reason for surfacing VMIs in the UI was that there is no point in hiding the VMIs from the UI.
+
+## UCs
+
+1. Debugging - If a user thinks that something isn’t right with the VMs and he wants to check what’s going on. When a VMI isn’t managed by a VM and it’s invisible in the UI but that VMI is actually still there because it’s impossible to delete VMIs (that’s a basic Kubernetes mechanism). This can happen by an orphan delete, or that a replica set isn’t used.
```

Comment:
```suggestion
1. Debugging - Previously if a user was trying to resolve an issue caused by a VMI that didn't have an owner it would not be possible to discover it in the UI. Including orphaned VMIs indicates to a user that there could be an issue that they need to address.
```